<launch>
  <!-- size of map, change the size in x, y, z according to your application -->
  <arg name="map_size_x" value="50.0"/>
  <arg name="map_size_y" value="50.0"/>
  <arg name="map_size_z" value=" 10.0"/>

  <arg name="init_x" value="0.0"/>
  <arg name="init_y" value="0.0"/>
  <arg name="init_z" value="1.0"/> 

    <!-- topic of your odometry such as VIO or LIO -->
  <arg name="odom_topic" value="/ardrone/ground_truth/odometry" />



<!-- gazebo仿真 -->
  <include file="$(find exploration_manager)/launch/gazebo_simulator.xml">
    <arg name="odometry_topic" value="$(arg odom_topic)" />
    <arg name="init_x" value="$(arg init_x)" />
    <arg name="init_y" value="$(arg init_y)" />
    <arg name="init_z" value="$(arg init_z)" />
  </include>



  
</launch>